SparkPost/support-docs
# EA-9270 - Add pages for competitive-tracker section of analyst

### What Changed

- Added pages for `competitive-tracker` pages and images.

### How To Test or Verify

- Verify `/analyst/competitive-tracker` pages exist and function (with images).
- Check pages match (except the intercom excerpts) - https://support.emailanalyst.com/en/collections/1525991-competitive-tracker

### PR Checklist
#### All PRs Checklist


#### Content Changes Checklist
